### ISSUE TITLE
Profile and accelerate geometric flow solver

### ISSUE THREAD

**Issue by sobolevnrm** [Electrostatics/apbs-pdb2pqr#220]
Probably best to fix #219 first.

**Comment by sobolevnrm** [comment]
Could be useful for #225 -- issue #225 should probably come first.

**Comment by keith923** [comment]
Is this a dupe of #235?

**Comment by lizutah** [comment]
Yes, if we think we've done all the acceleration necessary. Maybe other acceleration steps need to go in a separate issue?

**Comment by lizutah** [comment]
Closing since this is a duplicate of #235 and #225 should be done before profiling again.
